### PR TITLE
Make JavaScript work with LibreJS

### PR DESCRIPTION
**Pull request by dawidpotocki**
- [x] Works. LibreJS is happy.

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -31,3 +27,17 @@ function navSectionsClose(event) {
     navSection.open = !open;
   });
 }
+
+
+// Matomo
+var _paq = window._paq || [];
```
Comment (nitrohorse) [inline assets/js/main.js]: Doesn't matter too much but could we make this a bit more readable?
```javascript
// Matomo
var _paq = window._paq || [];
/* tracker methods like "setCustomDimension" should be called before "trackPageView" */
_paq.push(['trackPageView']);
_paq.push(['enableLinkTracking']);
(function() {
  var u = 'https://stats.privacytools.io/';
  _paq.push(['setTrackerUrl', u+'matomo.php']);
  _paq.push(['setSiteId', '1']);
  var d = document;
  var g = d.createElement('script');
  var s = d.getElementsByTagName('script')[0];
  g.type = 'text/javascript';
  g.async = true;
  g.defer = true;
  g.src = u+'matomo.js';
  s.parentNode.insertBefore(g,s);
})();
```

File: _includes/footer.html [hunk 2]
```diff
@@ -71,8 +71,12 @@ <h3>Copy URL and Description</h3>
   </div>
 
   <div class="copyright-text">
+    <a href="/about/javascript" data-jslicense="1">
+      <span class="fab fa-js fa-lg"></span>
+      JavaScript Licenses
+    </a>
```
Comment (nitrohorse) [inline _includes/footer.html]: Could we add a slight space here? `<br />`

File: about/javascript/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<h3>JavaScript Licenses</h3>
+<table id="jslicense-labels1" border="1">
+  <tr>
+    <td><a href="/assets/js/main.js">main.js</a></td>
+    <td><a href="http://creativecommons.org/publicdomain/zero/1.0/legalcode">CC0-1.0-only</a></td>
+  </tr>
+  <tr>
+    <td><a href="/assets/js/redirects.js">redirects.js</a></td>
+    <td><a href="http://creativecommons.org/publicdomain/zero/1.0/legalcode">CC0-1.0-only</a></td>
```
Comment (nitrohorse) [inline about/javascript/index.html]: I'm thinking for the creativecommons links we could use `https://`

**Comment by nitrohorse** [comment]
Super cool!

**Comment by JonahAragon** [comment]
Have you removed all the inline JS in this PR? We could probably update [the CSP](https://github.com/privacytoolsIO/privacytools.io/blob/master/nginx/010-headers.conf#L4) too.

**Comment by dawidpotocki** [comment]
> Have you removed all the inline JS in this PR? We could probably update [the CSP](https://github.com/privacytoolsIO/privacytools.io/blob/master/nginx/010-headers.conf#L4) too.

Well… I only found this in `scripts.html` and LibreJS does not inform me
about other stuff. So probably, yes.

**Pull request merged**